src/sync/app_credentials.cpp

```diff
@@ -16,9 +16,11 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
+#include "util.hpp"
```

Comment:
Is this includes and `<string>` needed?

Reply:
`<string>` isn't (probably leftover from some testing I did), but `util.hpp` is needed for parsing the credentials.

Reply:
This should be `#include "util/bson/bson.hpp"`.